traffic_portal/test/end_to_end/conf.json

```diff
@@ -10,7 +10,7 @@
     "defaultTimeoutInterval": 600000
   },
   "params": {
```

Comment:
CIAB would normally change the password for TP. But TP was unable to work on CIAB and had to be started on locally. None of the UI tests would pass until the password was changed.